data/3_deliveryMethodMeta.csv

```diff
@@ -79,18 +79,6 @@ UK_GUARANTEED_ROYAL_MAIL_SPECIAL_DELIVERY_1PM_SATURDAY_2500,1001,2500,2500,"Spec
 UK_GUARANTEED_ROYAL_MAIL_SPECIAL_DELIVERY_9AM_SATURDAY_50,0,50,50,"Special Delivery: Guaranteed by 9am Saturday"
```

Comment:
I don't need them for this change, but thought I would add in the changes I'm making rather then create a new PR